### PR TITLE
Pack setup to shell script (unix)

### PR DESCRIPTION
**Pull request by alexromanov**

### CODE REVIEW

**Comment by alexromanov** [comment]
https://github.com/Visual-Regression-Tracker/Visual-Regression-Tracker/issues/120

File: README.md [hunk 1]
```diff
@@ -44,6 +44,14 @@ New users and projects could be created via frontend app by default on http://lo
 
 ![Success setup](https://github.com/Visual-Regression-Tracker/Visual-Regression-Tracker/blob/master/public/docker_setup_creds.png)
 
+Using script (Unix based systems):
+
+1. Install [Docker](https://docs.docker.com/get-docker/)
+
```
Comment (pashidlos) [inline README.md:50]: gonna need one more step to download the script itself

File: vrt-install.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    echo "Visual Regression Tracker requires docker-compose. Check install docs here - $DOCKER_COMPOSE_URL"
+fi
+
+DOCKER_YML_URL=https://raw.githubusercontent.com/Visual-Regression-Tracker/Visual-Regression-Tracker/master/docker-compose.yml 
+DOCKER_ENV_URL=https://raw.githubusercontent.com/Visual-Regression-Tracker/Visual-Regression-Tracker/master/.env
+IMAGE_UPLOADS_FOLDER=imageUploads
+
+curl $DOCKER_YML_URL -o docker-compose.yml
+
+curl $DOCKER_ENV_URL -o .env
```
Comment (pashidlos) [inline vrt-install.sh:28]: env file gets overridden without saved changes
`docker-compose.yml` should not be an issue so far

File: vrt-install.sh [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    echo "Visual Regression Tracker requires Docker. Check install docs here - $DOCKER_INSTALL_URL"
+fi
+
+if [ ! -x "$(command -v docker-compose)" ]; then
+    echo "Visual Regression Tracker requires docker-compose. Check install docs here - $DOCKER_COMPOSE_URL"
+fi
+
+DOCKER_YML_URL=https://raw.githubusercontent.com/Visual-Regression-Tracker/Visual-Regression-Tracker/master/docker-compose.yml 
+DOCKER_ENV_URL=https://raw.githubusercontent.com/Visual-Regression-Tracker/Visual-Regression-Tracker/master/.env
+IMAGE_UPLOADS_FOLDER=imageUploads
```
Comment (pashidlos) [inline vrt-install.sh:24]: folder name is set in `.env` file 
could we take a value from there?

File: vrt-install.sh [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+curl $DOCKER_ENV_URL -o .env
+
+ENV_FILE=.env
+
+if [[ ! -z "$REACT_APP_API_URL" ]]; then 
+    IFS=':'
+    read -a url <<< "$REACT_APP_API_URL"
+    unset IFS
+    sed -ir "s~^[#]*\s*REACT_APP_API_URL=.*~REACT_APP_API_URL=$REACT_APP_API_URL~" $ENV_FILE
+    sed -ir "s~^[#]*\s*PORT=.*~port=${url[2]}~" $ENV_FILE
```
Comment (pashidlos) [inline vrt-install.sh:37]: `port` should be in upper case `PORT` with value for frontend port instead of backend port

File: vrt-install.sh [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    echo "Updated value for React App Api to $REACT_APP_API_URL"
+fi
+
+if [[ ! -z "$APP_FRONTEND_URL" ]]; then 
+    IFS=':'
+    read -a url <<< "$APP_FRONTEND_URL"
+    unset IFS
+    sed -ir "s~^[#]*\s*APP_FRONTEND_URL=.*~APP_FRONTEND_URL=$APP_FRONTEND_URL~" $ENV_FILE
+    sed -ir "s~^[#]*\s*APP_PORT=.*~APP_PORT=${url[2]}~" $ENV_FILE
```
Comment (pashidlos) [inline vrt-install.sh:46]: value should contain backend port default `4200`

File: README.md [hunk 6]
```diff
@@ -44,6 +44,14 @@ New users and projects could be created via frontend app by default on http://lo
...
+2. Make shell script executable: `$ chmod +x vrt-install.sh`
+
+3. Execute setuip script: `$ ./vrt-install.sh`
```
Comment (pashidlos) [inline README.md:53]: could we add example with params?

**Comment by dcrowe** [comment]
This PR has stalled for a few months. Are there any objections if I step in to finish it off?

**Comment by alexromanov** [comment]
@dcrowe  You can proceed with it. thanks.

**Pull request closed without merging**